### DISCUSSION TITLE
Screen rotation for linux

### DISCUSSION THREAD

**Discussion by java3000** [flareteam/flare-game#959] [Ideas]
There are several phones that run Linux OS, but not all of them are recognized by Flare as phones. Because of this, the screen doesn't automatically rotate to the largest possible horizontal position. If we could rotate the screen, it would allow us to use Flare on more devices.
Screenshot is how flare game looks on phone on portrait screen orientation. 
(white area - whole screen)
![photo_2024-05-27_13-26-57](https://github.com/flareteam/flare-game/assets/6387537/9d13b24f-1ad7-42af-b913-0e6e9d0abb00)

**Comment by dorkster** [comment]
You should be able to turn on Full Screen Mode in the Configuration menu to use your entire screen width. But I think the orientation should be handled by the window manager.

We probably don't need to default to windowed mode for new players at all anymore. It made sense when going full screen would change video modes, but we're just a border-less window these days.

Also, it might make sense to provide a command line flag to explicitly launch Flare as with `platform.is_mobile_device` set to true. When that flag is set, we pick some [defaults](https://github.com/flareteam/flare-engine/blob/master/src/Settings.cpp#L233-L244) that work better for touch-only devices.

**Reply by java3000** [reply]
Thank you, this helps me a lot!
